src/arrayMethodSort.js

```diff
@@ -4,9 +4,35 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareFunction2) {
+    let countr = 1;
+
+    while (countr !== 0) {
+      countr = 0;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        const current = this[i];
+        const next = this[i + 1];
+        const value = compareFunction(current, next);
+
+        if (value > 0) {
+          this[i] = next;
+          this[i + 1] = current;
+          countr++;
+        }
+      }
+    }
+
+    return this;
   };
 }
 
 module.exports = applyCustomSort;
+
+const compareFunction2 = (a, b) => {
+  if (a.toString() > b.toString()) {
```

Comment:
this function should also return  `0` when `a.toString() === b.toString()`.

Reply:
I'm sorry. But why? I have only one check for this output value. And for the task, I'm only interested in whether variable A is greater than variable B. If the value is 0, then it will behave the same as with a value of -1.

 I just made a review and left comments that can remove the check for _equals_ and _less_, so I have to defend my point of view. 😅

![image](https://user-images.githubusercontent.com/108505176/185191828-6c8f7a16-f18c-4d39-9cc1-c50c2bc52291.png)

Reply:
It is great that you could discuss and defend your opinion  👍 

Sorry, I missed 37 line of code.